images/proxysql/tests/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    oci    = { source = "chainguard-dev/oci" }
+    random = { source = "hashicorp/random" }
+    helm   = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+
+# we commented out the Helm tests since the Helm Chart uses the tag in labels, so, once we use pseudo tag, it gives an error like the following:
```

Comment:
Add this link https://github.com/dysnix/charts/blob/main/dysnix/proxysql/templates/_helpers.tpl#L41

And open an issue to track finding a fix for this.

Reply:
opened https://github.com/chainguard-images/images/issues/1065